src/components/form-draft/testing.vue

```diff
@@ -147,7 +166,15 @@ export default {
     ],
     // This text will be shown in ODK Collect when testing a Draft Form. {name}
     // is the title of the Draft Form.
-    "collectProjectName": "[Draft] {name}"
+    "collectProjectName": "[Draft] {name}",
+    "datasetsPreview": {
+      "title": "Datasets Preview",
```

Comment:
One other thought related to i18n: particularly for strings that aren't a full sentence, it can be helpful to add context for translators about how the string is used. Here, I would add a comment that explains why "Preview" is capitalized, something like:

```json5
// This is a title shown above a section of the page.
```

Our Transifex workflow will automatically add comments like that to Transifex for translators to see there.